ros2bag/ros2bag/verb/play.py

```diff
@@ -78,10 +78,20 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
                  '  pragmas: [\"<setting_name>\" = <setting_value>]'
                  'Note that applicable settings are limited to read-only for ros2 bag play.'
                  'For a list of sqlite3 settings, refer to sqlite3 documentation')
-        parser.add_argument(
+        clock_args_group = parser.add_mutually_exclusive_group()
+        clock_args_group.add_argument(
             '--clock', type=positive_float, nargs='?', const=40, default=0,
             help='Publish to /clock at a specific frequency in Hz, to act as a ROS Time Source. '
                  'Value must be positive. Defaults to not publishing.')
+        clock_args_group.add_argument(
+            '--clock-topics', type=str, default=[], nargs='+',
+            help='List of topics separated by spaces that will trigger a /clock update '
+                 'when a message is published on them'
+        )
+        clock_args_group.add_argument(
+            '--clock-topics-all', default=False, action='store_true',
+            help='Publishes an update on /clock immediately before each replayed message'
```

Comment:
@kylemarcey I think `--clock-topics-all` should be mutually exclusive with '--clock-topics' and '--clock' command line options.

Reply:
Sorry I missed that it is already mutually exclusive